mkdocs.yml

```diff
@@ -21,7 +22,7 @@ nav:
           - doc/visual-studio-code-setup.md
     - Internal code and APIs:
           - doc/working-with-bigints.md
-          - src/shared/math.md
+          - ... | flat | src/shared/*
```

Comment:
maybe you need to add a "flat" to the "src" in my comment above.

Reply:
Unfortunately that is not a viable solution because using 'flat' turns off the per-directory mkdocs index files, which are being used to hide the workbench pages in the "standard" case. So if we insert the `flat`, we will get the template visualizer doc page in production, which we don't want.

Reply:
Bottom line: if seamless formatting of the nav bar in workbench mode is a need-to-have, just let me know and I will dive into it. (I should handle because I definitely have the most background with mkdocs.)

Reply:
I see.  Well, in that case just leave it be and file an issue.  It's not high priority.

Reply:
No, really, Aaron, you have plenty to do, and I have been working with mkdocs way more. I do get the sense from both of you that you would really like it to look exactly the same except with the workbench visualizers. I will work on it, and if I don't get it by the time this is otherwise ready to merge, we can merge it and file an issue and I will keep working on it.

Reply:
Okay, @gwhitney is taking this over. [Apparently](https://henrywhitaker3.github.io/mkdocs-material-dark-theme/plugins/awesome-pages/) a `.pages` file can include a `title` attribute; could we use that to make the headings more readable?

Reply:
OK, I have it. Wasn't so hard. Sorry I just wasn't motivated before. Aaron, the change consists of adding one line each to mkdocs.yml and requirements.txt, diffs pasted below. If you prefer I push a commit, just let me know when.

```
diff --git a/mkdocs.yml b/mkdocs.yml
index d571946..17c498f 100644
--- a/mkdocs.yml
+++ b/mkdocs.yml
@@ -38,6 +38,7 @@ extra_javascript:
     - https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.0/MathJax.js?config=TeX-AMS-MML_HTMLorMML
 plugins:
     awesome-pages:
+        collapse_single_pages: true
         filename: mkdocs-index.yml
     search: {}
     semiliterate:
diff --git a/requirements.txt b/requirements.txt
index fd37b23..994ed0f 100644
--- a/requirements.txt
+++ b/requirements.txt
@@ -1,3 +1,4 @@
+mkdocs ~= 1.6
 mkdocs-semiliterate ~= 0.7
 mkdocs-awesome-pages-plugin ~= 2.8
 pymdown-extensions ~= 10.8
```